README.md

```diff
@@ -3,9 +3,35 @@
 
 # OpenQA Public cloud Helper
 
-PublicCloud-Watcher or `pcw` is a web app which monitors, displays and deletes resources on various Cloud Service Providers (CSPs). It identifies abandoned instances by searching for certain tags and performs cleanup tasks. It also deletes old custom uploaded images. The behavior differs per CSP.
-
-The fastest way to run PublicCloud-Watcher is via the provided containers, as described in the [Running a container](#running-a-container) section.
+PublicCloud-Watcher (PCW) is a web app which monitors, displays and deletes resources on various Cloud Service Providers (CSPs).
+PCW has two main flows :
+
+1. Update run ( implemented in [ocw/lib/db.py](ocw/lib/db.py) )
+ Concentrates on deleting VMs (in case of Azure Resource Groups) . Each update run scans accounts defined in configuration file and
```

Comment:
Why does an Update run concentrates on deleting VMs?

Reply:
I name this section like this because it coming from main function which handle it https://github.com/SUSE/pcw/blob/master/ocw/lib/db.py#L190 . Why this function named `update_run` no idea :) . I am fine to rename a function to any other name which you think suits better but I see this idea not directly related to current PR . In separate PR we may also update section name accordingly